backend/webserver/models.py

```diff
@@ -15,3 +16,61 @@ class FollowRequest(models.Model):
 class Follow(models.Model):
     follower = models.ForeignKey(Author, related_name='following_authors', on_delete=models.CASCADE)
     followee = models.ForeignKey(Author, related_name='followed_by_authors', on_delete=models.CASCADE)
+
+class Post(models.Model):
+
+    author = models.ForeignKey(Author,on_delete=models.CASCADE)
+
+    created_at = models.DateTimeField(verbose_name="date created",null=False, blank=False, default = datetime.now)
```

Comment:
```suggestion
    created_at = models.DateTimeField(verbose_name="date created", auto_now_add=True)
```
- [`null` is set to `False` by default](https://docs.djangoproject.com/en/4.1/topics/db/models/#field-options)
- [`blank` is set to `False` by default](https://docs.djangoproject.com/en/4.1/topics/db/models/#field-options)
- [`auto_now_add` will automatically set the timestamp when the object is first created](https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.DateField.auto_now_add)